src/renderer/components/Button/ButtonAction.tsx

```diff
@@ -2,11 +2,16 @@ import React from "react";
 import arrow from "../../assets/img/buttons/Backtab.svg";
 import add from "../../assets/img/buttons/Add.svg";
 import copyDefault from "../../assets/img/buttons/CopyDefault.svg";
+import ReactTooltip from "react-tooltip";
 
 export interface IActionButtonProps {
     onClick?: () => void;
 }
 
+export interface ICopyButtonProps extends IActionButtonProps {
+    clicked?: boolean;
```

Comment:
I don't think there is need for somebody else to set this. It would be better if that component uses state internally and resets after 1s to "Copy All" text